README.md

```diff
@@ -317,7 +335,37 @@ The `values` method returns an array of keys.
 
 ```javascript
 var objectType = new minim.ObjectType({ foo: 'bar' });
-var value = objectType.values() // ['bar']
+var values = objectType.values() // ['bar']
+```
+
+##### items
+
+The `items` method returns an array of key value pairs which can make iteration simpler.
+
+```js
+const objectType = new minim.ObjectType({ foo: 'bar' });
+
+for (let [key, value] of objectType.items()) {
+  console.log(key, value); // foo, bar
+}
+```
+
+### Element Registry
+
+Minim allows you to register custom types for elements. For example, if the element type name you wish to handle is called `category` and it should be handled like an array:
+
+```javascript
+var minim = require('minim');
+
+// Register your custom type
+minim.TypeRegistry.elementMap.category = minim.ArrayType;
```

Comment:
Regarding this, it almost feels like there needs to be something for registering.

```javascript
minim.TypeRegistry.register('category', minim.ArrayType);
```